draft-ietf-ohai-ohttp.md

```diff
@@ -235,7 +235,7 @@ Oblivious HTTP has limited applicability.  Many uses of HTTP benefit
 from being able to carry state between requests, such as with cookies
 ({{?RFC6265}}), authentication ({{Section 11 of HTTP}}), or even
 alternative services ({{?RFC7838}}).  Oblivious HTTP removes linkage
-at the transport layer, which must be used in conjunction with applications
+at the transport layer, and must be used in conjunction with applications
```

Comment:
The "which" here didn't parse correctly for me — it sounds like it is referring to the transport layer? Happy to see this cleaned up in other ways too.